X/XGBoost/build_tarballs.jl

```diff
@@ -93,10 +93,17 @@ for cuda_version in versions_to_build, platform in platforms
     if !isnothing(cuda_version) && !build_cuda
         continue
     end
-
-    augmented_platform = Platform(arch(platform), os(platform);
-        cuda=isnothing(cuda_version) ? "none" : CUDA.platform(cuda_version)
-    )
+    
+    # For Windows, we want to avoid adding cuda=none
+    # https://github.com/JuliaPackaging/Yggdrasil/issues/6911#issuecomment-1599350319
+    if os(platform) == "windows"
+        augmented_platform = deepcopy(platform)
```

Comment:
I think you want to do this for all platforms for which you don't build CUDA, i.e. anything different from x86_64-linux-gnu?

Reply:
I am dropping the tags from all the other platforms where we can't build CUDA binaries.